### DISCUSSION TITLE
Global Script

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#997] [Q&A (Help)] [answered]
I've a project based on Laravel 11 and Backpack 6 working fine for many months
I have always used a chatbot (tawk.to) that I set up using the Glabal Scrips placed in ui.php
```
    // --------------
    // GLOBAL SCRIPTS
    // --------------

    // JS files that are loaded in all pages, using Laravel's asset() helper
    'scripts' => [
        'js/tawktASH.js',
        // 'js/example.js',
        // 'https://unpkg.com/vue@2.4.4/dist/vue.min.js',
        // 'https://unpkg.com/react@16/umd/react.production.min.js',
        // 'https://unpkg.com/react-dom@16/umd/react-dom.production.min.js',
    ],
```
After one of the latest updates made with composer update it no longer works and looking at the source of the page I see that the js/tawktASH.js module is not loaded at all and there's no error in browser console

What can it be ?
Thanks for help

**Comment by jcastroa87** [comment]
Hello @blondie63,

Have you tried loading any other scripts to see if they work? Also, can you confirm if the script is available at the specified route?

Please provide more details so I can assist you further.

Cheers.

**Reply by blondie63** [reply]
My script never moved from the folder and it worked until recently, I also tried to write an incorrect file name but I see no errors on the browser console, this means there is no attempt to load it, what you think ?
what script could I load to do a test?
Thanks

**Reply by jcastroa87** [reply]
Can you share you versions please.

I made a test on my side:
```
### PHP VERSION:
8.3.6

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dba, dom, hash, FFI, fileinfo, filter, ftp, gd, gmp, json, iconv, SPL, session, intl, standard, ldap, mbstring, mysqlnd, mysqli, pcntl, exif, imap, PDO, pdo_mysql, pdo_pgsql, pdo_sqlite, pgsql, Phar, posix, random, readline, redis, Reflection, imagick, shmop, SimpleXML, soap, sockets, sodium, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, Zend OPcache

### LARAVEL VERSION:
11.8.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.4
backpack/crud: 6.7.14
backpack/devtools: 3.1.5
backpack/generators: v4.0.5
backpack/pro: 2.2.2
backpack/theme-tabler: 1.2.10
```

And is working normal

<img width="583" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/44643608/85805dfe-1382-4857-ad3a-8b30cd89490e">

<img width="716" alt="image" src="https://github.com/Laravel-Backpack/community-forum/assets/44643608/8b211c15-287d-492c-819b-4a445f408b5a">

Is trying to load example.js

Cheers.

**Reply by blondie63** [reply]
Hi @jcastroa87 this is my version & console info:
### PHP VERSION:
8.3.7

### PHP EXTENSIONS:
Core, date, libxml, openssl, pcre, sqlite3, zlib, bcmath, bz2, calendar, ctype, curl, dba, dom, hash, FFI, fileinfo, filter, ftp, gd, gettext, gmp, json, iconv, intl, SPL, ldap, mbstring, session, standard, odbc, pcntl, exif, mysqlnd, PDO, pdo_dblib, pdo_mysql, PDO_ODBC, pdo_pgsql, pdo_sqlite, pgsql, Phar, posix, pspell, random, readline, Reflection, mysqli, shmop, SimpleXML, soap, sockets, sodium, sysvmsg, sysvsem, sysvshm, tidy, tokenizer, xml, xmlreader, xmlwriter, xsl, zip, xdebug, Zend OPcache

### LARAVEL VERSION:
11.8.0.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.3.4
backpack/crud: 6.7.14
backpack/devtools: 3.1.5
backpack/editable-columns: 3.0.9
backpack/generators: v4.0.5
backpack/language-switcher: 2.0.0
backpack/pro: 2.2.0
backpack/settings: 3.1.1
backpack/theme-coreuiv2: 1.2.3
backpack/theme-tabler: 1.2.10

<img width="585" alt="Screenshot 2024-05-28 alle 15 51 47" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/009cb00d-faac-4303-b267-45d20078a9f1">
<img width="578" alt="Screenshot 2024-05-28 alle 15 52 11" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/fb674bbf-6a7b-4b19-8f9d-98c3e8708bed">

**Reply by jcastroa87** [reply]
What theme are you using in the tests?

**Reply by jcastroa87** [reply]
you didnt override this file? /resources/views/ui/inc/scripts.blade.php, can you show the content of this file please. You can compare with: https://github.com/Laravel-Backpack/CRUD/blob/main/src/resources/views/ui/inc/scripts.blade.php

Please try to clean basset cache:

```
php artisan basset:clear
php artisan basset:cache 
 ```
 
Let me know if one of this work for you.

Cheers.

**Reply by blondie63** [reply]
I'm using coreuiv2
I've tried the two basset:.. cmds
I no have override of scripts.blade.php
:(

**Reply by jcastroa87** [reply]
Can you verify if override any view who load scripts.blade.php, maybe a comment line or something, because i start a project from scratch and is working normal.

Cheers.

**Reply by blondie63** [reply]
@jcastroa87 I searched my entire project folder but there is only one scripts.blade.php file and it is in the folder:
myapp/vendor/backpack/crud/src/resources/views/ui/inc/

**Reply by jcastroa87** [reply]
Hello @blondie63 please let me know if you had any luck finding any differences in the views or any other Backpack files. If you found no difference, did you try testing this behavior on a fresh Backpack installation?

Cheers.

**Reply by blondie63** [reply]
Please excuse the delay..
I confirm that I have not found any differences in the Backpack files and in this application I have not made any changes but only some composer updates! But I tried to use the same .js module on a new project and it works!! What else can I look for?

**Reply by jcastroa87** [reply]
Hello @blondie63 

I dont know if you test in the same server, but this look like cache issue or maybe some miss setup in the production server.

If this work normal in a new updated project and there is no any diff in files, so we have not som much alternatives.

Did you try to downgrade to the last working version?

Cheers.

**Reply by blondie63** [reply, marked as answer]
Hi @jcastroa87 i've a found a solution..
I create an override to vendor/backpack/theme-coreuiv2/resources/views/inc/head.blade.php adding this line:
@basset('js/tawktASH.js')

and i've commented the line in ui.php
    // JS files that are loaded in all pages, using Laravel's asset() helper
    'scripts' => [
        //'js/tawktASH.js',
    ],
    
  Do you know why it works like this?
